appng-core/src/main/java/org/appng/core/model/ThymeleafProcessor.java

```diff
@@ -130,7 +159,7 @@ public String processWithTemplate(Site applicationSite) throws InvalidConfigurat
 			File config = new File(platformCache, ResourceType.XML.getFolder());
```

Comment:
This line can be deleted?

Reply:
Yes, I removed it and pushed the change on the branch